DeliveryService/Filters/CustomActionAttribute.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Filters;
+using System;
+using System.IO;
+using System.Text;
+
+namespace DeliveryService.API.Filters
+{
+    public class CustomActionAttribute : IActionFilter
+    {
+        public void OnActionExecuted(ActionExecutedContext context)
+        {
+        }
+
+        public void OnActionExecuting(ActionExecutingContext context)
+        {
+            var bodyStr = "";
```

Comment:
u can remove it 
"var bodyStr = "";"

and use it
"var bodyStr = reader.ReadToEnd();"